src/node_url.h

```diff
@@ -63,6 +68,8 @@ class BindingData : public SnapshotableObject {
 
   void UpdateComponents(const ada::url_components& components,
                         const ada::scheme::type type);
+
+  static v8::CFunction fast_can_parse_;
```

Comment:
Stylistic issue: static member fields don't have the trailing underscore, that's only for instance fields.
```suggestion
  static v8::CFunction fast_can_parse;
```

Reply:
this pattern is used elsewhere, see
https://github.com/nodejs/node/blob/40d808f1fbaa7b3666e5837a1b4336947180fac5/src/timers.h#L56-L59 and
https://github.com/nodejs/node/blob/40d808f1fbaa7b3666e5837a1b4336947180fac5/src/node_process.h#L91-L92

Reply:
Standards are slipping everywhere, it seems. ^_^